build.sbt

```diff
@@ -55,15 +55,15 @@ lazy val cassandra =
     .settings(Defaults.itSettings)
     .settings(Dependencies.cassandra)
     .dependsOn(core)
-    .dependsOn(coreTest % "test->compile;it->test")
+    .dependsOn(coreTest % "test->test;it->test")
```

Comment:
Here is a special case. Cassandra project doesn't have test sources, only integration tests.  Without adding `test->test`, the pom.xml don't get it declared.